docs/buttons/button-shape.mdx

```diff
@@ -12,61 +12,91 @@ components: ShapeButton
 <WindowEditor codeString="import { ShapeButton } from 'dd360-ds/Buttons'" />
+<WindowEditor codeString="import ShapeButton from 'dd360-ds/ShapeButton'" />
 
 ### Usage
 
 #### BaseCircleButton
 
 To use the BaseCircleButton follow this example:
 
-<ContainerComponentDoc className="gap-y-2">
+<ContainerComponentDoc>
     <ShapeButton.BaseCircleButton />
 </ContainerComponentDoc>
 
-<WindowEditor codeString="<ShapeButton.BaseCircleButton />" />
+<WindowEditor codeString="import { ShapeButton } from 'dd360-ds'
+
+<ShapeButton.BaseCircleButton />" />
+
+
+#### Variants BaseCircleButton
+
+The prop variant can only be used with the BaseCircleButton, it has 2 variants: "square" and "circle".
+By default it is "circle".
+
+<ContainerComponentDoc className="flex align-center items-center gap-x-3">
+    <div className="flex flex-col align-center items-center">
+        <Text className='flex mb-1'>Square</Text>
+        <ShapeButton.BaseCircleButton variant='square' />
+    </div>
+    <div className="flex flex-col align-center items-center">
+        <Text className='flex mb-1'>Circle</Text>
+        <ShapeButton.BaseCircleButton variant='circle' />
+    </div>
+</ContainerComponentDoc>
+
+<WindowEditor codeString="import { ShapeButton } from 'dd360-ds'
+
+<ShapeButton.BaseCircleButton variant='square' />
+<ShapeButton.BaseCircleButton variant='circle' />" />
 
 #### Square Button
 
 To use the SquareButton follow this example:
 
-<ContainerComponentDoc className="gap-y-2">
+<ContainerComponentDoc>
     <ShapeButton.SquareButton />
 </ContainerComponentDoc>
 
-<WindowEditor codeString="<ShapeButton.SquareButton />" />
+<WindowEditor codeString="import { ShapeButton } from 'dd360-ds'
+
+<ShapeButton.SquareButton />" />
 
 #### Circle Button
 
 To use the CircleButton follow this example:
 
-<ContainerComponentDoc className="gap-y-2">
+<ContainerComponentDoc>
     <ShapeButton.CircleButton />
 </ContainerComponentDoc>
 
 <WindowEditor
     codeString="import { ShapeButton } from 'dd360-ds'
- <ShapeButton.CircleButton Icon={() => <DynamicHeroIcon icon='AcademicCapIcon' className='text-blue-700 w-6' />}  />"
+
+ <ShapeButton.CircleButton />"
 />
 
 #### ShapeButton with custom icon
 
 In addition, you can customize the buttons using custom icons:
 
-<ContainerComponentDoc className="gap-y-2">
-    <ShapeButton.BaseCircleButton Icon={() => <DynamicHeroIcon icon="AcademicCapIcon" className="text-blue-700 w-6" />}/>
+<ContainerComponentDoc>
+    <ShapeButton.BaseCircleButton icon={<DynamicHeroIcon icon="AcademicCapIcon" className="text-blue-700 w-6" />}/>
 </ContainerComponentDoc>
 
 <WindowEditor codeString="import { ShapeButton } from 'dd360-ds'
 import DynamicHeroIcon from 'dd360-ds/DynamicHeroIcon'
 
-<ShapeButton.BaseCircleButton Icon={() => <DynamicHeroIcon icon='AcademicCapIcon' className='text-blue-700 w-6' />} />" />
+<ShapeButton.BaseCircleButton 
+&nbsp;&nbsp;icon={<DynamicHeroIcon icon='AcademicCapIcon' className='text-blue-700 w-6' />} 
```

Comment:
```suggestion:
&nbsp;&nbsp;icon={<DynamicHeroIcon icon='AcademicCapIcon' className='text-blue-700 w-6' />} 
```

Clarification: the props of the code inside the prop "codeString" of the WindowEditor component, are written with single quotes, because if they are written with double quotes, they are broken since they are inside the prop "codeString".